services/movie.py

```diff
@@ -1,33 +1,34 @@
 from django.db.models import QuerySet
-
+from django.db import transaction
 from db.models import Movie
 
 
 def get_movies(
-    genres_ids: list[int] = None,
-    actors_ids: list[int] = None,
+        genres_ids: list[int] | None = None,
+        actors_ids: list[int] | None = None,
+        title: str | None = None,
 ) -> QuerySet:
```

Comment:
You should specify inner type of `QuerySet`
```suggestion
) -> QuerySet[Movie]:
```